src/data-workspace/data-details-sidebar/update-limits.js

```diff
@@ -46,31 +46,25 @@ function UpdateLimits({
                     <LimitsInput
                         {...minField.input}
                         label={limitInputLabelsByName.min}
-                        error={!!minField.meta.error}
+                        error={minField.meta.touched && !!minField.meta.error}
+                        meta={minField.meta}
```

Comment:
do you want to pass this? It seems like it's not beeing used in LimitsInput component?

Reply:
Nop, this is remnants from another way to do this... Thanks!